crates/solver/src/driver.rs

```diff
@@ -529,9 +536,10 @@ impl Driver {
             .rate_settlements(solver_settlements, &external_prices, gas_price)
             .await?;
         tracing::info!(
-            "{} settlements passed simulation and {} failed",
+            "{} settlements passed simulation and {} failed for auctionId: {}",
```

Comment:
I think consistently using either `auction`, `auction id` or `auctionId` would also be nice for grep-ability.
The 3 new/updated log messages all use slightly different wording.